aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -175,34 +177,24 @@ impl VersionedValue {
     fn read_latest_committed_value(
         &self,
         next_idx_to_commit: TxnIndex,
-    ) -> Result<u128, MVAggregatorsError> {
+    ) -> Result<AggregatorValue, MVAggregatorsError> {
         use AggregatorEntry::*;
 
         self.versioned_map
             .range(0..next_idx_to_commit)
             .next_back()
-            .map_or(
-                self.base_value.ok_or(MVAggregatorsError::NotFound),
+            .map_or_else(
+                || self.base_value.clone().ok_or(MVAggregatorsError::NotFound),
```

Comment:
Why clone?

Reply:
I cannot consume self.base_value, I need to clone it to return it?